docs/papers-selection.md

```diff
@@ -212,9 +212,28 @@ Internal software quality, e.g. the quality of code, has great impact on the ove
 
 ### Review
 
-| Paper_id | Name  | Definition | Qualitative/Quantitative | Targeted to SW | Reviewer  | Comment |
-| :------: | :---: | :--------: | :----------------------: | :------------: | :-------: | :-----: |
-| 82       |       |            |                          |                | MV        |         |
+| Paper_id | Name  | Definition | Qualitative/Quantitative | Targeted to SW | Reviewer  | Comment   |
+| :------: | :---: | :--------: | :----------------------: | :------------: | :-------: | :-------: |
+| 82       | n/a   | n/a        | n/a                      | n/a            | MV        | See below |
+
+* *MV* - this paper indeed describes the "EMISQ" approach to internal(*) software quality assessment
+  ("internal" as in "quality aspects that are primarily visible to software architects, developers and testers").
+  The overall idea is to combine static analysis tools with expert input, for which EMISQ provides a framework.
+  It provides a corresponding quality model, but seemingly without describing the quality criteria in significant
+  detail. For this, the reader is referred to the ISO 9126 quality model and the following static analysis tools
+  (primarily aimed at C++ and Java code, apparently):
+
+  - CPD: https://pmd.github.io/latest/pmd_userdocs_cpd.html
+  - FindBugs: http://findbugs.sourceforge.net
+  - JLint: http://jlint.sourceforge.net/
+  - JMetric: https://sourceforge.net/projects/jmetric/
+  - PC-Lint: https://gimpel.com/
+  - PMD: https://pmd.github.io/
+  - QJPro: http://qjpro.sourceforge.net/
+
+  (*) "Internal" as in "quality aspects that are primarily visible to software architects, developers and testers".
+  "External" quality aspects "are noticeable by the users of a software product". I think this is indeed a useful
+  distinction.
```

Comment:
Are there any of these further explained, so we can add them to the table?
It looks like the ISO 9126 standard is definitely something we'll want to add.

Reply:
I've now added the main features of internal/external aspects as described in the paper ([fbe94d0](https://github.com/eosc-sq/ensure-sq/pull/2/commits/fbe94d059ef4cd1f95a873d119af5272c00df00e)).

They also list of (internal) quality metrics in Figure 4 (page 6). I can sure add them as well in this entry, but is this valuable without further description about what they mean?

Regarding ISO 9126 standard: I presume that one was already on our radar, and that it will be processed/reviewed separately (i.e. not as a part of this paper's review?).

Reply:
- Without context, probably not that much, I agree.
- The standard should be reviewed separately. Otherwise it would not be fair for us to ask to go through each of the references.
Thanks!